Code/JavaWrappers/ChemReactions.i

```diff
@@ -72,6 +73,17 @@ static std::string ReactionToSmarts(ChemicalReaction &rxn) {
 static std::string ReactionToRxnBlock (const ChemicalReaction &rxn) {
   return RDKit::ChemicalReactionToRxnBlock(rxn);
 };
+/*
+static RDKit::ROMol *ReduceProductToSideChains(RDKit::ROMOL_SPTR product,
+                                               bool addDummyAtoms=true) {
+  return RDKit::reduceProductToSideChains(product, addDummyAtoms);
+};
+*/
+static RDKit::ROMol *ReduceProductToSideChains(RDKit::ROMol *product,
```

Comment:
I believe that this will leak unless you also have a `%newobject` call present to tell SWIG that Java owns the new object